azure-pipelines/secure-development-tools.yml

```diff
@@ -1,5 +1,14 @@
 steps:
 
+ ### Scan binaries with Windows Defender
+- task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
+  displayName: 'Scan with Windows Defender'
+  inputs:
+    TreatSignatureUpdateFailureAs: Warning
```

Comment:
If we're going to get this all the time, can we not treat it as a warning either? I really want our pipelines to usually "pass" without showing a warning indicator all the time.

Reply:
The available options are: Error, Standard, Warning. Standard is the same as Warning.

Reply:
Are we required to run this task? If it's going to cause every build result to be a warning, I'm not happy. I'd want to file a bug against them at the _very_ least.

Reply:
Removing in favor of the existing Microbuild Signing task that covers the requirement